runbotics/runbotics-orchestrator-ui/src/src-app/Actions/types.ts

```diff
@@ -19,6 +19,10 @@ export enum CloudPath {
     SITE = 'site',
 }
 
+export enum Patterns {
+    VARIABLE_INPUT = '^[a-zA-Z0-9_]*$',
```

Comment:
replace * with + (as * means 0-infinity, and + means 1-infinity)
now I guess it can be completely empty 

also imo rename VARIABLE_INPUT to VARIABLE_NAME